### ISSUE TITLE
Support reading remote vp files

### ISSUE THREAD

**Issue by peterdesmet** [adokter/bioRad#544]
Cf. how `readr::read_csv()` allows to read remote csv files, it would be useful that `read_vpfiles()` did the same. This especially useful to create reproducible examples:

```R
library(bioRad)
read_vpfiles("https://aloft.s3-eu-west-1.amazonaws.com/baltrad/hdf5/bejab/2022/08/19/bejab_vp_20220819T235500Z_0x9.h5")
```

**Comment by bart1** [comment]
I have looked at some stage into this. Reading `h5` files from remote is not easy. I would avoid going there. If you are referring to the `csv` it is indeed easily done without temporary files using `vroom` or `readr` (the former allows to modify the request using curl which can be useful for authentication).

**Comment by bart1** [comment]
I just saw your example was with an `h5` file there fore my feedback

**Comment by peterdesmet** [comment]
I agree that reading csv files remotely is quite easy, can be implemented in `read_vpts()` (#551). That would help to write shorter reproducible examples. The typical workflow would still be to download first (#553) and read second.

**Comment by peterdesmet** [comment]
_Discussed at the Amsterdam 2026 coding sprint_

@bart1: we're likely not going to read vp (h5) files remotely. We should focus on reading vpts files.
